pyoozie/tags.py

```diff
@@ -613,6 +615,87 @@ def _xml(self, doc, tag, text, on_next, on_error):
         return doc
 
 
+class Serial(_WorkflowEntity):
+    """Sequence of actions to execute (implemented by chaining actions and 'OK' transitions)"""
+
+    def __init__(self, *actions, **kwargs):
+        # type: (*_WorkflowEntity, **_WorkflowEntity) -> None
+        super(Serial, self).__init__(xml_tag=None, on_error=kwargs.get(str('on_error')))
+        self.__actions = tuple(copy.deepcopy(actions))  # type: typing.Tuple[_WorkflowEntity, ...]
```

Comment:
Anything that's permitted to be edited can be, but most properties are locked down and most constructors copy. To split up object creation you need to start from the leaves inwards with this API.